apis/core/lib/handlers/csv-source.ts

```diff
@@ -1,10 +1,11 @@
 import asyncMiddleware from 'middleware-async'
 import * as labyrinth from '@hydrofoil/labyrinth/resource'
+import { parse } from 'content-disposition'
+import clownface from 'clownface'
+import { is } from 'type-is'
```

Comment:
If you default-import

```suggestion
import typeis from 'type-is'
```